tsconfig.json

```diff
@@ -24,10 +24,13 @@
     "strict": true,
     "strictNullChecks": true,
     "target": "ESNext",
-    "tsBuildInfoFile": "node_modules/.cache/.tsbuildinfo",
+    "tsBuildInfoFile": "node_modules/.cache/tsbuildinfo/build",
```

Comment:
Was this not working previously?

Reply:
this _was_ working, but is clobbered by the cypress/tsconfig.json value. Now, they'll both have their own caches :)